gobblin-data-management/src/test/java/gobblin/data/management/copy/CopyableFileTest.java

```diff
@@ -30,6 +32,7 @@
 import gobblin.configuration.ConfigurationKeys;
 import gobblin.util.PathUtils;
 
+import static org.mockito.Mockito.*;
```

Comment:
I was going for mockito's suggestion on usage:
http://mockito.org/
This is a test and a static import, so I am not sure it should be covered so strictly by Gobblin's import format. However, I am happy to switch it if needed.